tests/utils/compare.js

```diff
@@ -1,15 +1,26 @@
 /* global XMLHttpRequest, expect */
 
-function loadBinaryResource (url) {
+function loadBinaryResource (url, unicodeCleanUp) {
   const req = new XMLHttpRequest()
   req.open('GET', url, false)
    // XHR binary charset opt by Marcus Granado 2006 [http://mgran.blogspot.com]
-  req.overrideMimeType('text\/plain; charset=x-user-defined')
+  req.overrideMimeType('text\/plain; charset=x-user-defined');
   req.send(null)
   if (req.status !== 200) {
     throw new Error('Unable to load file')
   }
-  return req.responseText
+      var responseText = req.responseText;
+    var responseTextLen = req.responseText.length;
+  var StringFromCharCode = String.fromCharCode;
```

Comment:
Seem to be some formatting issues around here

Reply:
He copy pasted it from my PR #1460 . And tbh, after few hours of try and error i was just happy, that it works :). formatting was not my primary goal :)

Reply:
Hah no worries, yeah just flagging :)